packages/app/pages/class/[cid]/queue/[qid].tsx

```diff
@@ -166,15 +148,14 @@ export default function Queue({}: QueueProps) {
     question: Question,
     status: QuestionStatus
   ) => {
-    const q = await API.questions.update(queueId, question.id, {
+    const q = await API.questions.update(Number(qid), question.id, {
       status: status,
     });
 
     if (q) {
-      // fetch updated question list
-      getQuestions();
-
       // update helping state if none left
+      //temporary
```

Comment:
Make sure to use `mutate` here as well in order to display updated questions: https://github.com/vercel/swr#mutation-and-post-request